L/libwebp/build_tarballs.jl

```diff
@@ -49,9 +53,11 @@ dependencies = [
     Dependency("Giflib_jll"),
     Dependency("JpegTurbo_jll"),
     Dependency("libpng_jll"),
-    Dependency("Libtiff_jll"; compat="4.3.0"),
+    Dependency("Libtiff_jll"; compat="4.7.0"),
```

Comment:
After #10181 can do
```suggestion
    Dependency("Libtiff_jll"; compat="4.7.1"),
```
and a few more packages will be available for aarch64-freebsd and riscv64-linux